site/docs/plugins/i18n.md

```diff
@@ -391,13 +396,20 @@ interface SessionData {
 
 type MyContext = Context & SessionFlavor<SessionData> & I18nFlavor;
 
+const bot = new Bot<MyContext>(""); // <-- put your bot token here
+
+// Be awared, in Deno Deploy service,
+// we can only load fluent scripts after init I18n.
 const i18n = new I18n<MyContext>({
   defaultLocale: "en",
   useSession: true, // whether to store user language in session
-  directory: "locales", // Load all translation files from locales/.
+
+  // DOES NOT work in Deno Deploy
+  directory: "locales",
 });
 
-const bot = new Bot<MyContext>(""); // <-- put your bot token here
+// uncomment to asynchronously load translations in Deno Deploy
```

Comment:
```suggestion
// Translation files loaded this way works on Deno Deploy, too.
```

Reply:
I don't understand there is `too` in the end sentence. It's not like the `directory` method work with Deno Deploy right?

Reply:
The too means that it will work on Deno Deploy alongside other Deno environments.

Reply:
I understand differently. I thought that there is other way that supposes to mention before that can work on Deno Deploy which we do not. Imho, the previous comment contains more meaning and clearer then this one. It gave a clear instruction on what to do and for what case.